tests/admin/controller/test_controller.py

```diff
@@ -3123,288 +3130,260 @@ def test_create_integration(self):
+    #             ("short_name", "nypl"),
+    #             (Configuration.FEATURED_LANE_SIZE, "20"),
+    #             (Configuration.MINIMUM_FEATURED_QUALITY, "0.9"),
+    #             (Configuration.WEBSITE_URL, "https://library.library/"),
+    #             (Configuration.DEFAULT_NOTIFICATION_EMAIL_ADDRESS, "email@example.com"),
+    #             (Configuration.DEFAULT_FACET_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME,
+    #              FacetConstants.ORDER_AUTHOR),
+    #             (Configuration.ENABLED_FACETS_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME + "_" + FacetConstants.ORDER_AUTHOR,
+    #              ''),
+    #             (Configuration.ENABLED_FACETS_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME + "_" + FacetConstants.ORDER_RANDOM,
+    #              ''),
+    #         ])
+    #         flask.request.files = MultiDict([])
+    #         response = self.manager.admin_settings_controller.libraries()
+    #         eq_(response.status_code, 200)
+    #
+    #     library = get_one(self._db, Library, uuid=library.uuid)
+    #
+    #     eq_(library.uuid, response.response[0])
+    #     eq_(library.name, "The New York Public Library")
+    #     eq_(library.short_name, "nypl")
+    #
+    #     # The library-wide settings were updated.
+    #     def val(x):
+    #         return ConfigurationSetting.for_library(x, library).value
+    #     eq_("https://library.library/", val(Configuration.WEBSITE_URL))
+    #     eq_("email@example.com", val(Configuration.DEFAULT_NOTIFICATION_EMAIL_ADDRESS))
+    #     eq_("20", val(Configuration.FEATURED_LANE_SIZE))
+    #     eq_("0.9", val(Configuration.MINIMUM_FEATURED_QUALITY))
+    #     eq_(FacetConstants.ORDER_AUTHOR,
+    #         val(Configuration.DEFAULT_FACET_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME)
+    #     )
+    #     eq_(json.dumps([FacetConstants.ORDER_AUTHOR, FacetConstants.ORDER_RANDOM]),
+    #         val(Configuration.ENABLED_FACETS_KEY_PREFIX + FacetConstants.ORDER_FACET_GROUP_NAME)
+    #     )
+    #
+    #     # The library-wide logo was not updated and has been left alone.
+    #     eq_("A tiny image",
+    #         ConfigurationSetting.for_library(Configuration.LOGO, library).value
+    #     )
+
+    # def test_library_delete(self):
+    #     library = self._library()
+    #
+    #     with self.request_context_with_admin("/", method="DELETE"):
+    #         self.admin.remove_role(AdminRole.SYSTEM_ADMIN)
+    #         assert_raises(AdminNotAuthorized,
+    #                       self.manager.admin_settings_controller.library,
+    #                       library.uuid)
+    #
+    #         self.admin.add_role(AdminRole.SYSTEM_ADMIN)
+    #         response = self.manager.admin_settings_controller.library(library.uuid)
+    #         eq_(response.status_code, 200)
+    #
+    #     library = get_one(self._db, Library, uuid=library.uuid)
+    #     eq_(None, library)
```

Comment:
It looks like all this commented-out code was moved to test_library, so it can be removed from here.